src/character.cpp

```diff
@@ -8416,15 +8415,15 @@ int Character::height() const
 {
     switch( get_size() ) {
         case creature_size::tiny:
-            return init_height - 100;
+            return init_height / 2; // 50% original
```

Comment:
I think this is a little cleaner (and removes the need for comments, because the code is more self-documenting). It isn't necessary to make these changes, however.
```suggestion
            return init_height * 0.5;
```

Reply:
Yeah, I considered doing that. Wanted to avoid unnecessary int/float conversions. But I agree it's much more readable this way.